bokeh/io/notebook.py

```diff
@@ -439,8 +439,8 @@ def load_notebook(resources: Resources | None = None, verbose: bool = False,
 
     if not hide_banner:
         if resources.mode == 'inline':
-            js_info = 'inline'
-            css_info = 'inline'
+            js_info : str|List[str] = 'inline'
+            css_info: str|List[str] = 'inline'
```

Comment:
Is there a way to avoid this?

Reply:
They joys of  variable scoping in Python combined with static typing. This should be rewritten as follows:
```py
js_info: str | List[str]
css_Info: str | List[str]
if ...:
    js_info = 'inline'
    css_info = 'inline'
else:
    js_info = [...]
    css_Info = [...]

Reply:
https://github.com/bokeh/bokeh/pull/11597/commits/1403ed71eba395bde34f5276cf5c495b31d81de7

Reply:
Maybe a MyPy bug or limitation, but it fails with this change. I will revert for now.